kong/constants.lua

```diff
@@ -159,6 +159,9 @@ local constants = {
     CASSANDRA = {
       MIN = "3.0",
       DEPRECATED = "2.2",
+    },
+    REDIS = {
+      MIN = "5.0"
```

Comment:
Is this only true when ´username` is used? Otherwise we do support less than that?

Reply:
Fero and I discussed on slack and we agreed to not include the redis support minimum version check. That can be added back at a later time (3.0).

Reply:
Thanks for the discussion Alan; consider this reverted!